package.json

```diff
@@ -48,9 +49,9 @@
     "sort-json-keys": "^1.0.3",
     "ts-jest": "^27.1.3",
     "ts-node": "^10.5.0",
-    "ts-xlsx": "^0.0.11",
-    "typescript": "^4.5.5"
-  }, 
+    "typescript": "^4.5.5",
```

Comment:
this should be under `devDependencies`

Reply:
@bjpirt One for you

Reply:
I think we need to have a bit of a sort of deps vs devdeps - there isn't really a "production" in this case anyway since it all runs in a GitHub action